2020/index.html

```diff
@@ -0,0 +1,176 @@
+<!doctype html>
+<html lang="en">
+<head>
+
+	  <!-- Charset attribute for the webpage -->
+    <meta charset="utf-8">
+
+    <!-- Meta, stylesheets and scripts -->
+    <meta name="viewport" content="width=device-width, initial-scale=1, shrink-to-fit=yes">
+    <link rel="stylesheet" href="../styles/2020/main.css">
+    <link rel="icon" href="../img/assets/logo2020.png">
+
+    <!-- Title -->
+    <title>Hacklahoma 2020</title>
+
+    <!-- Description for search engines -->
+    <meta name="description" content="Hacklahoma. February 8-9 2020. Come create with us at our third annual hackathon!" />
+
+    <!-- Facebook pixel stuff -->
+    <meta property="og:title" content="HACKLAHOMA">
+    <meta property="og:type" content="website">
+    <meta property="og:image" content="https://hacklahoma.org/img/facebookCardImage.png">
+    <meta property="og:image:width" content="1200">
+    <meta property="og:image:height" content="630">
+    <meta property="og:image:type" content="image/png">
+    <meta property="og:url" content="https://hacklahoma.org">
+    <meta property="og:description" content="Hacklahoma. February 8-9 2020. Come create with us at our third annual hackathon!">
+
+    <!-- Facebook Pixel Code -->
+    <script>
+        !function(f,b,e,v,n,t,s){if(f.fbq)return;n=f.fbq=function(){n.callMethod?
+        n.callMethod.apply(n,arguments):n.queue.push(arguments)};if(!f._fbq)f._fbq=n;
+        n.push=n;n.loaded=!0;n.version='2.0';n.queue=[];t=b.createElement(e);t.async=!0;
+        t.src=v;s=b.getElementsByTagName(e)[0];s.parentNode.insertBefore(t,s)}(window,
+        document,'script','https://connect.facebook.net/en_US/fbevents.js');
+        fbq('init', '1555823664723200'); // Insert your pixel ID here.
+        fbq('track', 'PageView');
+    </script>
+    <noscript>
+      <img height="1" width="1" style="display:none" src="https://www.facebook.com/tr?id=1555823664723200&ev=PageView&noscript=1"/>
+    </noscript>
+
+    <!-- FB Pixel that tracks when people visit our website -->
+    <script>
+    fbq('track', 'ViewContent');
+    </script>
+</head>
+
+<body>
+
+  <!-- Header -->
+  <section>
+    <div class="header-container">
+      <div class="header-title">
+        <div class="header-text">
+          <h1>Hacklahoma</h1>
+          <a class="button" href="https://docs.google.com/forms/d/e/1FAIpQLScBR51amjtUec0MJSzVzPiop8uZ-a4kqCp5ow6OIsYIQSEHIQ/viewform">Stay updated</a>
```

Comment:
People will want to know the date on this page. I'd probably just put 

<h2>February 8-9 2020</h2>

here or something like that with some padding.